examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/snarkjs/snarkjs.service.ts

```diff
@@ -29,30 +32,25 @@ export class SnarkjsCircuitService implements ICircuitService {
       preparedInputs,
       pathToCircuit,
       pathToProvingKey,
+      pathToWitnessCalculator,
+      pathToWitnessFile,
     );
 
     this.witness.proof = proof;
 
     this.witness.publicInputs = publicInputs;
 
-    // TODO: stack Error: Cannot find module 'zeroKnowledgeArtifacts/circuits/workstep1/workstep1_circuit_verification_key.json'
-    // from '../src/bri/zeroKnowledgeProof/services/circuit/snarkjs/snarkjs.service.ts'
-    // this.witness.verificationKey = await import(pathToVerificationKey);
-
+    this.witness.verificationKey = JSON.parse(
+      fs.readFileSync(pathToVerificationKey, 'utf8'),
+    );
     return this.witness;
   }
 
   public async verifyProofUsingWitness(witness: Witness): Promise<boolean> {
-    const isVerified = await snarkjs.groth16.verify(
+    const isVerified = await snarkjs.plonk.verify(
```

Comment:
why are we changing from groth16 to plonk here?

Reply:
During BRI-3/Interop calls, we decided to use plonk because it does not require circuit-specific phase 2 contribution, significantly reducing the setup time.